gallery/src/styles/NxList/NxListExamples.tsx

```diff
@@ -7,18 +7,21 @@
 import React from 'react';
 import { GalleryExampleTile } from '../../gallery-components/GalleryTiles';
 
+import NxListClickableExample from './NxListClickableExample';
+import NxListWithActionsExample from './NxListWithActionsExample';
+import NxListErrorExample from './NxListErrorStateExample';
+
 const NxListSimpleCode = require('!!raw-loader!./NxListDefaultExample.html').default,
-    NxListClickableCode = require('!!raw-loader!./NxListClickableExample.html').default,
+    NxListClickableCode = require('!!raw-loader!./NxListClickableExample').default,
     NxListMultiLineCode = require('!!raw-loader!./NxListMultiLineExample.html').default,
     NxListBulletedCode = require('!!raw-loader!./NxListBulletedExample.html').default,
-    NxListWithActionsCode = require('!!raw-loader!./NxListWithActionsExample.html').default,
-    NxListDefinitionListCode = require('!!raw-loader!./NxListDefinitionListExample.html').default,
+    NxListWithActionsCode = require('!!raw-loader!./NxListWithActionsExample.tsx').default,
     NxListEmptyCode = require('!!raw-loader!./NxListEmptyExample.html').default,
-    NxListErrorStateCode = require('!!raw-loader!./NxListErrorStateExample.html').default;
+    NxListErrorStateCode = require('!!raw-loader!./NxListErrorStateExample.tsx').default;
 
 const NxListsExamples = () =>
   <>
-    <GalleryExampleTile title="NX Simple List Example"
+    <GalleryExampleTile title="NX Basic List Example"
                         id="nx-list-simple-example"
                         htmlExample={NxListSimpleCode}
                         codeExamples={NxListSimpleCode}>
```

Comment:
Description below is outdated: mentions two lists when there's only one

Reply:
> Description below is outdated: mentions two lists when there's only one

https://github.com/sonatype/sonatype-react-shared-components/pull/141/commits/76906a1d34d5ad3164803a2b0470fa1ce854781d